presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -563,20 +563,20 @@ Procedures
 
   Create an empty partition in the specified table.
 
-* ``system.sync_partition_metadata(schema_name, table_name, mode, case_sensitive)``
+* ``system.sync_partition_metadata(schema_name, table_name, mode, optional case_sensitive)``
 
   Check and update partitions list in metastore. There are three modes available:
 
   * ``ADD`` : add any partitions that exist on the file system, but not in the metastore.
   * ``DROP``: drop any partitions that exist in the metastore, but not on the file system.
   * ``FULL``: perform both ``ADD`` and ``DROP``.
 
-  The ``case_sensitive`` argument is optional. The default value is ``true`` for compatibility
+  Default value for optional ``case_sensitive`` argument is ``true`` for compatibility
   with Hive's ``MSCK REPAIR TABLE`` behavior, which expects the partition column names in
   file system paths to use lowercase (e.g. ``col_x=SomeValue``). Partitions on the file system
   not conforming to this convention are ignored, unless the argument is set to ``false``.
 
-* ``system.drop_stats(schema_name, table_name, partition_values)``
+* ``system.drop_stats(schema_name, table_name, optional partition_values)``
```

Comment:
I am not aware of this syntax being used for optional values anywhere else in the docs. Is this okay @electrum ? In other places we just show the different signatures from what I know